src/main/java/net/mcreator/ui/blockly/BlocklyEditorToolbar.java

```diff
@@ -146,12 +146,19 @@ public BlocklyEditorToolbar(MCreator mcreator, BlocklyEditorType blocklyEditorTy
 			add(component);
 		}
 
-		for (var component : extraComponents) {
+		for (JComponent component : extraComponents) {
 			add(component);
 		}
 
 		add(Box.createHorizontalGlue());
 
+		JButton cleanup = L10N.button("blockly.templates." + blocklyEditorType.registryName() + ".cleanup");

```

Comment:
Since text will likely not change for different editors and is currently the same, I would suggest using same translation key for all cases